components/Card.js

```diff
@@ -1,7 +1,48 @@
 import React from "react";
+import Card from "react-bootstrap/Card";
 
-const Card = () => {
-	return <div>Card</div>;
-};
+class Block extends React.Component {
+	render() {
+		return (
+			// have one shadow white and one black right behind it looking like a border
+			<div className='flex justify-center'>
+				<Card
+					style={{
+						width: "14rem",
+						backgroundColor: "black",
+						borderRadius: 25,
+						boxShadow: "12px -13px white, 16px -17px black",
+					}}
+				>
+					<Card.Body>
+						<br />
+						<div className='flex justify-evenly '>
+							<div className='columns-1 text-blue-500'>
+								<Card.Title>09</Card.Title>
+								<Card.Title>21</Card.Title>
+								<Card.Title>22</Card.Title>
+							</div>
 
-export default Card;
+							<div className='columns-1'>
+								<div className='text-blue-500'>
+									<Card.Title>General Meeting</Card.Title>
```

Comment:
Regarding the content, put it into props, so that we can use the component as a template